queries.sql

```diff
@@ -131,4 +131,87 @@ ORDER BY num_animals DESC
 LIMIT 1;
 
 
```

Comment:
## Project Requirement :arrow_down:
![image](https://user-images.githubusercontent.com/93039359/221071689-657b1a60-e99b-46df-a903-cfa706b5c718.png)

- [ ] You have screenshots of your Select queries in your pull request but it would be nice if you add screenshots with the results of the queries because now I cannot see any queried data (results) in your screenshots :thinking: Kindly add new screenshots with queried data results :+1:

Reply:
I have updated the PR Summary to include the images of all the queries performed in the database as requested by Microverse. 

Thank you for your feedback